setup.py

```diff
@@ -23,6 +23,16 @@ def check_output(*popenargs, **kwargs):
 
 import sys
 
+if os.environ.get('COVERAGE','') == 'true':
```

Comment:
Can this just be any non-empty string?  setting it to 'true' exactly is too easy to get wrong.